source/scenes/DayOf/scenes/Maps/styles.scss

```diff
@@ -0,0 +1,102 @@
+.map-image {
+  width: 80vw;
+  height: auto;
+  padding-bottom: 20px;
+}
+
+.tab-title {
+  color: #ffffff;
+  font-family: Montserrat, Verdana, serif;
+  font-size: 44px;
+  font-weight: bold;
+  margin-bottom: 28px;
+  margin-top: 48px;
+}
+
+.react-tabs__tab-panel,
+.react-tabs__tab {
+  color: #fd6f6d;
+  font-family: Montserrat, Verdana, serif;
+  font-weight: 600;
+  font-size: 18px;
+  padding: 0;
+}
+
+.react-tabs {
+  overflow: scroll;
+  height: 100vh;
+  position: absolute;
+  top: 0;
+  width: 80vw;
+}
+::-webkit-scrollbar {
+  width: 0px; /* remove scrollbar space */
+  background: transparent; /* optional: just make scrollbar invisible */
+}
+
+.react-tabs__tab-panel--selected {
+  clear: both;
+}
+
+.tab-underline {
+  border-bottom: #fd6f6d 5px var(--border-style-bottom);

```

Comment:
It's better to use Sass variables than `CSS` variables but I'm only seeing one instance of use so the variable probably isn't needed in the first place.

Reply:
Didn't use Sass variable since it didn't have the desired behavior. Since I'm using `react-tabs` and not an `<a>` link, the selection/active wouldn't apply to the `<p className="tab-underline">` but to the `<tab>` element. Sass var will affect all or none of the `<p>` elements based on the scope of the var. Using a css var will cascade down from the `<tab>` down to the `<p>` element, with the state depending on the whether or not the react-tab is selected.

Reply:
Or I can just used a selector like `.react-tabs__tab--selected > .tab-underline { border-bottom: #fd6f6d 5px solid; }`, which works just fine for the job